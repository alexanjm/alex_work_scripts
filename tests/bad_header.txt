age	weight
alex	24	195
ryan	22	190
kimani	23	160
