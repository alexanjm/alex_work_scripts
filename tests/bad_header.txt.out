Column Name	weight
alex	195
ryan	190
kimani	160
